testing/google-cloud-appengine-flex-custom/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM gcr.io/google_appengine/jetty
+ADD google-cloud-appengine-flex-custom-0.10.1-alpha-SNAPSHOT.war $JETTY_BASE/webapps/root.war
```

Comment:
Yes, we will. And it should be updated here (I forgot to update it after pull). I don't think it is possible to make it relative to current version (the only way i see is to use that update_pom_version.sh script). This file is a Docker file, which knows nothing about maven, java and any other build/versioning stuff we use.